<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareView</class>
 <widget class="QWidget" name="CompareView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="lblLeft">
         <property name="text">
          <string>Left</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tbContainerLeft" native="true"/>
       </item>
       <item>
        <widget class="QHexEdit" name="hexEditLeft" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_1">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QLabel" name="lblRight">
         <property name="text">
          <string>Right</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tbContainerRight" native="true"/>
       </item>
       <item>
        <widget class="QHexEdit" name="hexEditRight" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="ActionWidget" name="actionWidgetLeft" native="true"/>
     </item>
     <item>
      <widget class="ActionWidget" name="actionWidgetRight" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>qhexedit/qhexedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActionWidget</class>
   <extends>QWidget</extends>
   <header>actionwidget/actionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
